### DISCUSSION TITLE
Tool Tip intercepts click to minimize file diff

### DISCUSSION THREAD

**Discussion by jchesshirAspire** [community/community#67203] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm new to posting in the GitHub community, so I'm wondering which conversation category to put this in.  I'm thinking either "Discussions" or "Projects and Issues".  I'm primarily starting the conversation in the General category to be pointed to the proper place:

> When I am looking through a set of file diffs and I move my mouse in an upward direction to minimize a file with diffs that start below the very top of the page, a tooltip invariably intercepts the click.
> 
> ![image](https://github.com/community/community/assets/141282351/17dd1a58-688b-4a6f-9c2f-a88193ebb5ce)
> 
> Could the tooltip be moved to below the box to avoid getting in the way of other functionality?
> 

Which category would you say this this topic belongs in?

**Comment by ZhengLinLei** [comment]
I don't have this issue, but you can go to F12 and remove it from `html` or create a script and bookmark it, so you don't need to open F12 each time just only clicking the bookmark you can hide the tooltip

![imagen](https://github.com/community/community/assets/77938581/e03709af-e791-46c1-95e4-6adc5a1ea0d9)

**Reply by jchesshirAspire** [reply]
I'm not looking for a hack to hide the problem from just myself.  I'm looking for the forum that GitHub developers review to get them to consider fixing this problem for everyone.
